docs/usage/java.md

```diff
@@ -23,10 +23,52 @@ Renovate does not support:
 - Android projects that require extra configuration to run (e.g. setting the Android SDK)
 - Gradle versions prior to version 5.0.
 
+## Gradle Wrapper
+
+Renovate can update the [Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) of a project.
+
+This includes the source declaration inside the `gradle/wrapper/gradle-wrapper.properties`
+as well as accompanied files such as `gradlew`, `gradlew.bat`, and `gradle/wrapper/gradle-wrapper.jar`.
+
 ### How It Works
 
-Renovate uses a plugin to search and extract versions from projects.
-Once the Gradle plugin has detected the dependencies, lookups and updating will be performed like usual with datasources and direct patching of files.
+Renovate extracts the Gradle Wrapper version used from the `distributionUrl` inside the `gradle-wrapper.properties`.
+Once the version is determined, Renovate will look for newer versions from the `gradle-version` datasource.
+Renovate will then invoke the Gradle Wrapper to update itself,
+[as recommended by Gradle](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper).
+
+For the extraction to work, the `distributionUrl` must point to
+a file of type .zip,
+which includes the version in its name, and
+defines one of the official distribution types (bin, all).
```

Comment:
```suggestion
For the extraction to work, the `distributionUrl` must point to a file of type `.zip`, which includes the version in its name, and defines one of the official distribution types (bin, all).
```

Use one sentence per line.
Use monospaced styling for the `.zip` file extension.